lib/src/builder.dart

```diff
@@ -128,19 +129,29 @@ class AggregateTestBuilder extends Builder {
         final path =
             p.relative(id.path, from: p.dirname(buildStep.inputId.path));
         imports.writeln("import '$path' as $prefix;");
-        mains.writeln("  $prefix.main();");
+        mains.add("  $prefix.main();");
       }
     }
 
     // Don't generate an empty aggregate test.
     if (imports.isEmpty) return;
+    if (_config.randomizeOrderingSeed != null) {
+      var seed = _config.randomizeOrderingSeed.toLowerCase() == 'random'
+          ? Random().nextInt(4294967295)
+          : int.parse(_config.randomizeOrderingSeed);
+      log.info(
+          'Shuffling test order with --test-randomize-ordering-seed=$seed');
```

Comment:
Unfortunately the `--test-randomize-ordering-seed` flag here doesn't help much since we're not actually using the test runner's randomization feature, so this might confuse users if they then try to use it.
```suggestion
      log.shout(
          'Shuffling test order with `randomize_ordering_seed: $seed`');
```